sourcegraph/sourcegraph
# web: fix blob table spacing

Fixes blob single-row table spacing issue. 

| Before | After |
| -- | -- |
|<img width="954" alt="Screenshot 2022-06-23 at 11 38 48" src="https://user-images.githubusercontent.com/25318659/175258731-4da65682-ad0c-4fa6-8823-3bce3dcf2660.png"> | <img width="954" alt="Screenshot 2022-06-23 at 11 39 37" src="https://user-images.githubusercontent.com/25318659/175258438-ef2feefb-8ffb-48bc-a119-18dfad974343.png"> |
|![image](https://user-images.githubusercontent.com/25318659/175258850-d9fd5191-8a51-4d98-9dcb-88f8b32b79b1.png)| <img width="954" alt="Screenshot 2022-06-23 at 11 45 43" src="https://user-images.githubusercontent.com/25318659/175258623-e38d7aef-b91d-4c32-91dc-6d398d5d9fd5.png"> |


## Test plan
- run `sg web standalone`
- open blob view of the file consisting og one line
- ensure it has proper spacing both with and without decorations column



## App preview:

- [Web](https://sg-web-taras-yemets-fix-codeview-table.onrender.com)
- [Storybook](https://5f0f381c0e50750022dc6bf7-axqclnodeq.chromatic.com)

Check out the [client app preview documentation](https://docs.sourcegraph.com/dev/how-to/client_pr_previews) to learn more.
